support/runtime-upgrade/index.js

```diff
@@ -0,0 +1,57 @@
+const { ApiPromise, WsProvider, Keyring } = require('@polkadot/api');
+
+var argv = require('minimist')(process.argv.slice(2));
+const fs = require('fs');
+
+// Create a promise API instance of the passed in node address.
+async function createPromiseApi(nodeAddress) {
+    const wsProvider = new WsProvider(nodeAddress);
+    const api = await new ApiPromise({ provider: wsProvider });
+    await api.isReady;
```

Comment:
I think all API methods await this for us. I think you can just `await new ApiPromise`.

Reply:
That's very strange, I haven't had to do that anywhere else.